### PR TITLE
Activate full size test

### PR DESCRIPTION
**Pull request by skrakau**
Add full-size test and activate for AWS testing.

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] If necessary, also make a PR on the [nf-core/mag branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/mag)

### CODE REVIEW

File: .github/workflows/awsfulltest.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@ name: nf-core AWS full size tests
 on:
   release:
     types: [published]
+  # TODO: just for testing, remove again
+  push:
+    branches:
+      - dev
+  pull_request:
```
Comment (ggabernet) [inline .github/workflows/awsfulltest.yml:12]:
```suggestion
```

Comment (ggabernet) [reply .github/workflows/awsfulltest.yml:12]: Hi Sabrina, the AWS tests will not run on pull requests as they require the organization secrets. So we can remove this here :smile:

**Review by ggabernet — APPROVED** [review]

**Comment by skrakau** [comment]
Thanks @ggabernet :)

**Comment by skrakau** [comment]
@ggabernet could you merge it? For me it says the required tests have to pass.

**Comment by ggabernet** [comment]
we're just testing the AWS tests 👍

**Comment by skrakau** [comment]
Thank you! :)

**Pull request merged**